python/ray/data/read_api.py

```diff
@@ -1755,6 +1761,109 @@ def create_connection():
     )
 
 
+@PublicAPI(stability="alpha")
+def read_databricks_uc_tables(
+    *,
+    warehouse_id: str,
+    table: Optional[str] = None,
+    query: Optional[str] = None,
+    catalog: Optional[str] = None,
+    schema: Optional[str] = None,
+    parallelism: int = -1,
+    ray_remote_args: Optional[Dict[str, Any]] = None,
+) -> Dataset:
+    """
+    Read from a Databricks UC table or Databricks SQL execution result that queries
+    from Databricks UC tables.
+    If it is not called in databricks runtime, you need to set environment varaibles
+    'DATABRICKS_HOST' and 'DATABRICKS_TOKEN' firstly.
```

Comment:
Does "it" mean "API" here?
```suggestion
    If this API isn't called in Databricks runtime, set the 'DATABRICKS_HOST' and 'DATABRICKS_TOKEN'
    environment variables.
```